plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageDatabaseUserInterface.java

```diff
@@ -203,7 +203,7 @@ protected void performDefaults()
 
     @Override
     public boolean isValid() {
-        return super.isValid() &&
+        return super.isValid() && clientTimezone != null &&
             (Arrays.stream(clientTimezone.getItems()).anyMatch(s -> s.equals(clientTimezone.getText())));
```

Comment:
```suggestion
        return super.isValid() && clientTimezone != null && ArrayUtils.contains(clientTimezone.getItems(), clientTimezone.getText());
```

How come the `clientTimezone` can be `null`?